OneBlink.SDK/models/FormSubmissionEventConfiguration.cs

```diff
@@ -2,6 +2,11 @@
 using System.Collections.Generic;
 namespace OneBlink.SDK.Model
 {
+    //Being kept right now due to backwards compatability, will be deleted upon next breaking change.
+    public class FormSubmissionEventConfigration : FormSubmissionEventConfiguration

```

Comment:
Wouldnt this just band aid it though? Pretty sure the 'real' function on line 10 needs to have the typo no?